astropy/time/core.py

```diff
@@ -11,6 +11,7 @@
 import copy
 import operator
 from datetime import datetime, date, timedelta
+import threading
```

Comment:
Nitpick... 😅 
```suggestion
import threading
from datetime import datetime, date, timedelta
```

Reply:
What is the general policy for ordering imports? I had been aiming for alphabetical.

Reply:
There is no official policy but most of the files have `import` happen first, followed by `from`, but within each block, I think they are mostly alphabetical, though @astrofrog once threatened to re-arrange by line length. 😆

Reply:
We have a config for isort (with the length sort, #8235), but the PR reordering all files was not merged ( #8250).